src/bci_build/package/micro/README.md.j2

```diff
@@ -2,12 +2,36 @@
 {% include 'badges.j2' %}
 
 ## Description
-This image is similar to Minimal but without the RPM package manager.
+The `bci-micro` image is smaller than `bci-minimal` because it
+does not include the RPM package manager, only the RPM database.
+
 The primary use case for the image is deploying static binaries produced
-externally or during multi-stage builds. As there is no straightforward
-way to install additional dependencies inside the container image,
-we recommend deploying a project using the Minimal image only
-when the final build artifact bundles all dependencies and has no
-external runtime requirements (like Python or Ruby).
+externally or during multi-stage builds.
+
+
+## Usage
+
+As there is no straightforward way to install additional
```

Comment:
Do we really need this?

> As there is no straightforward way to install additional dependencies inside the container image

If you remove this sentence it does not affect the overall idea of this section, plus, I can install packages with rpm, it is not as straightforward as zypper, but it is still usable and easy.

Reply:
rpm is not included in the micro image